# Generated by Django 3.2.4 on 2021-06-20 02:13

from django.conf import settings
from django.db import migrations, models
import pica.validators


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('pica', '0016_auto_20210618_0918'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='topik',
            name='topik2peserta',
        ),
        migrations.AddField(
            model_name='topik',
            name='topik2user',
            field=models.ManyToManyField(related_name='user2topik', to=settings.AUTH_USER_MODEL, verbose_name='Person in charge'),
        ),
        migrations.AlterField(
            model_name='departemen',
            name='name',
            field=models.CharField(blank=True, max_length=50, null=True, verbose_name='Nama Divisi/Departemen'),
        ),
        migrations.AlterField(
            model_name='forum',
            name='nama_forum',
            field=models.CharField(blank=True, max_length=100, null=True, validators=[pica.validators.validate_empty]),
        ),
    ]
